### ISSUE TITLE
Icon accessiblity in 2.6.1

### ISSUE THREAD

**Issue by SSPJ** [keepassxreboot/keepassxc#5455]
## Overview
It looks like the decision on Material Design icons was already made, so I'm not sure there is any actionable fix here.

However, previously, I was able to easily distinguish between icons even with poor eyesight. Now I cannot, so this seems like a regression.

If I could make the icons about 3x - 4x larger, I would be able to tell them apart, but I cannot find any option to do this.

I know the contributors who switched things over worked really hard on it and they did it so that things would look nicer, but it feels incredibility frustrating as a disabled person when previously functional software breaks.

Especially with something like a password manager, where rolling back to a previous version isn't a feasible solution.

## Expected Behavior
Familiar icons to be retained OR an option to switch back to the old icons OR a preview period for UI updates to gather a11y feedback.

## Actual Behavior
Launched keepassxc after a routine upgrade and spent 30 minutes searching documentation for how to make the icons usable again.

## Context
KeePassXC - Version 2.6.1
Revision: 9a35bba

Operating System: Ubuntu Linux
Desktop Env: XMonad
Windowing System: X11

**Comment by droidmonkey** [comment]
Which icons are you referring to exactly? We made the toolbar icons bigger, and the icons in the menus have not changed in size (those are default size to the OS platform). The icons on entries are also bigger. Please provide a screenshot pointing to the exact icons that give you trouble.

**Comment by droidmonkey** [comment]
Our entire UI is also screen reader accessible and can assist you with learning the new layout. All input fields and interactive icons are named / described for screen readers.

**Comment by SSPJ** [comment]
These are the icons:

![Screenshot_2020-09-20_22-13-10](https://user-images.githubusercontent.com/15718530/93730618-e99a9e00-fbb8-11ea-93a2-19d66c3ea0f4.png)

I believe you that they are using some setting outside the application, but the odds of me finding that setting are pretty slim. I've tried adjusting the "icon size" in Xfce4 and it had no effect.

Thanks for the tip about the screen reader. I do not need one for most of my applications, but I am glad to learn the descriptions are there for those who do. Given that I use keepassxc daily (thank you, btw!), I expect I will be able to memorize the location of the new icons, whether or not I can visually distinguish them.

**Comment by droidmonkey** [comment]
Those icons are sized with the toolbar size itself. Make sure you don't have compact mode enabled (View menu). We can add a setting to increase the toolbar size, but it is larger than previous versions already. If you hover over each icon a tool tip will appear showing you a description.

**Comment by SSPJ** [comment]
I did have compact mode on and I turned it off, but it didn't make the icons any bigger. It only put more white space around them. :) In the docs you link, the icons do look bigger.

![Screenshot_2020-09-20_22-52-32](https://user-images.githubusercontent.com/15718530/93731810-346ae480-fbbe-11ea-9a67-5c56760e5261.png)

I would love a setting to adjust these icons, but maybe I am the only person experiencing this issue?

**Comment by droidmonkey** [comment]
This is probably an issue with your desktop env and qt version. Try using our appimage or snap to help control that.

**Comment by xavupkt** [comment]
The new icons are UGLY, I can't work like this. I would like to have option to switch back to the original look. Should I downgrade the version? @droidmonkey

**Comment by phoerious** [comment]
You mean the pixelated low-res 16x16px icons from 2002? They won't be coming back.

**Comment by xavupkt** [comment]
> You mean the pixelated low-res 16x16px icons from 2002? They won't be coming back.
I see. Some people would like them back, cause we're used to that look. OR if the developers can put dark outline around them or something.

**Comment by promi** [comment]
We (my girlfriend and I) also find the new icons to be hard to distinguish. Especially the three icons for adding, editing and deleting look very similar on the first glimpse. They are all shaped like a circle, mazbe you should get fird of the circles and just have an +, pencil and x sign.

Also with the old icons there were colors to help to tell them apart.

**Comment by gagarine** [comment]
1) By default the toolbar should have text. 
2) Toolbar should be editable.
3) Some buttons may not be necessary. For example "copy X" could be replaced by a "double click" on the field directly in the list or a small "copy" icon in the entry details beside each field. I'm not sure a save/open button is really important (you can use the shortcut or menu and it's better to ask for save on quit + auto-save in case of crash).

![Screenshot 2021-02-19 at 19 06 15](https://user-images.githubusercontent.com/164074/108543613-994d7680-72e5-11eb-838e-53452a643f13.png)

**Comment by phoerious** [comment]
The toolbar is far too wide with text, so it's not on by default. An editable toolbar is out of scope at the moment. Qt has a widget for that, but it's super basic and doesn't allow for things such as embedded search fields.

**Comment by gagarine** [comment]
> The toolbar is far too wide with text

Agree, not ideal.

It was also not possible to implement the MacOS native toolbar because of the limitation of Qt. Personally I would prefer to hide the toolbar all together, but I would miss the search fields. Perhaps another approach could be to move the search field out of the toolbars to add it just before the entries list? But that's another bugs certainly.

Those 3 icons seem the most problematic:
![Screenshot 2021-02-19 at 21 22 46](https://user-images.githubusercontent.com/164074/108557284-a45dd200-72f8-11eb-895a-bf2ddf21fa8d.png)

Replacing the X delete with a trash:
![Screenshot 2021-02-19 at 21 23 17](https://user-images.githubusercontent.com/164074/108557357-c3f4fa80-72f8-11eb-93af-072f5f1f2082.png)

Not using a circle for edit but a square
![image](https://user-images.githubusercontent.com/164074/108557559-11716780-72f9-11eb-8cf7-676ba1a6ad80.png)

**Comment by promi** [comment]
>     3\. Some buttons may not be necessary. For example "copy X" could be replaced by a "double click" on the field directly in the list or a small "copy" icon in the entry details beside each field. I'm not sure a save/open button are important (you can use the classic ctrl-s shortcut or menu.).

The “copy X” buttons are a primary feature for me, please don't remove them from the toolbar.

**Comment by droidmonkey** [comment]
I have no plans to make any changes to the tool bar

**Comment by 4501khwy** [comment]
@droidmonkey Late to the conversation, but I think keepassxc does have an icon accessibility issue....depending on what format is being used.  The screenshot below shows the difference between the appimage (left) and the snap format (right).  The icon scaling is why I stay with the appimage.
![Screenshot from 2021-03-27 16-18-24](https://user-images.githubusercontent.com/81489868/112734331-73864380-8f1b-11eb-9d7b-5c966acaeb34.png)
Both instances are shown in "Classic" theme (Compact mode is unchecked).
System details are: Ubuntu 20.04.2, Wayland, Scaling set to 200% in gnome Settings, screen resolution is 3840x2160

**Comment by droidmonkey** [comment]
That's an issue with snaps in general.
